scripts/test/unit-with-coverage

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+for pkg in $(go list ./... | grep -v /vendor/); do
+    ./scripts/test/unit \
+        -cover \
+        -coverprofile=profile.out \
+        -covermode=atomic \
+        ${pkg}
+    
+    if test -f profile.out; then
```

Comment:
The file isn't copied. The file is appended to the global coverage file yes. That's what it's doing.